pkg/blockbuilder/tsdb.go

```diff
@@ -256,6 +259,15 @@ func (b *TSDBBuilder) newTSDB(tenant tsdbTenant) (*userTSDB, error) {
 	userID := tenant.tenantID
 	userLogger := util_log.WithUserID(userID, b.logger)
 
+	udb := &userTSDB{
+		userID: userID,
+	}
+
+	userLimit := b.limits.MaxGlobalSeriesPerUser(userID)
+	if userLimit <= b.applyGlobalSeriesLimitUnder {
```

Comment:
I realize this is a temporary measure, but `Under` doesn't match with `<=`. It's really an `AtMost`. Perhaps `applyGlobalSeriesLimitMax`?

Reply:
With the reword to `applyMaxGlobalSeriesPerUser...`, using `Max` as a suffix again sounds odd. How about `applyMaxGlobalSeriesPerUserBelow`? And put in comments that it is inclusive.